about.html

```diff
@@ -12,15 +12,15 @@
     <div class="c_about">
         <p class="p1_about">Hi I'm a web developer, I'm from Ecuador, I'm 26 years old.
             I'm learning to make web pages and in here public a lot of things that I learn</p>
-        <img class="img_about"
+        <img class="img_about" role="img"
```

Comment:
Role = image in an image tag is redundant.
For accessibility, I recommend adding an Alt attribute